src/decision/justify_info.h

```diff
@@ -0,0 +1,58 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Justification info.
+ */
+
+#include "cvc5_private.h"
+
+#ifndef CVC5__DECISION__JUSTIFY_INFO_H
+#define CVC5__DECISION__JUSTIFY_INFO_H
+
+#include "context/cdo.h"
+#include "expr/node.h"
+#include "prop/sat_solver_types.h"
+
+namespace cvc5 {
+
+/** A pair indicating a node and its desired value */
+using JustifyNode = std::pair<TNode, prop::SatValue>;
+
+/**
+ * Information concerning a single formula in the justification strategy.
+ */
+class JustifyInfo
+{
+ public:
+  JustifyInfo(context::Context* c);
+  ~JustifyInfo();
+  /** set */
+  void set(TNode n, prop::SatValue desiredVal);
```

Comment:
Can we make `n` const?

Reply:
I think the option would be to change this to `const Node& n` right?